mining.md

```diff
@@ -107,7 +107,7 @@ Chain selection is a crucial component of how the Filecoin blockchain works. Eve
 
 ### Block Validation
 
-The block structure and serialization is detailed in [the datastructures spec](data-structures.md#block). Check there for details on fields and types.
+The block structure and serialization is detailed in [the datastructures spec](data-structures.md#block). Check there for details on fields and types. All references to `VaidateSignature` must implement the `Signature` interface found [here](https://github.com/filecoin-project/specs/blob/master/signatures.md).
```

Comment:
This addition is confusing. Why not just say that signature validation is covered in detail in the signature spec?

Reply:
This is a good point. I've been mulling over how to best represent that `ValidateSignature` must use the verification method in signatures.md. The current phrasing is confusing b/c it should read: "`Block` implements the `Signature` interface" and `ValidateSignature` --> `Block.Verify(SignedMessage_Block, PublicKey)`. in this case, the internal block messages would be represented as a single value (hashed), `SignatureBytes` would be the actual signature of the message and `Address` would be the address of the block proposer.

Reply:
we also need a place that implements verification of the aggregated signatures, e.g. a method like `VerfiyAggregates([]Message, SignatureBytes) bool` for use inside the block validation.